docs/troubleshooting/pool_operations_hung.md

```diff
@@ -0,0 +1,25 @@
+# Pool Operation Hung Due to Bad Disk
+
+As of now, cStor scans all the devices on the node when it tries to import the pool in case of a pool manager pod
+restart. If any of the devices(even the devices that are not part of the cStor pool) are bad and is not responding the 
+command issued by cStor keeps on waiting and is stuck. As a result of this, pool manager pod is not able to issue any 
+more command in order to reconcile the state of cStor pools or even perform the IO for the volumes that are placed on 
+that particular pool. 
+
+A fix in this is being worked upong and more details can be found in the following link:
+https://github.com/openebs/openebs/pull/2935
+
+# How to recover from this situation?
+
+Now there could be two different situations :
+
+1. The device that has gone bad is actually a part of the cStor pool on the node. If this is the case
+then see [Block Device Replacement](../tutorial/cspc/tuning/tune.md) in this tutorial. This tutorial is for
+`mirror` RAID but similar approach can be followed for other RAID configurations like `raidz1` and `raidz2`. 
+
+**Note:** Block device replacement is not supported for `stripe` raid configuration. 
```

Comment:
Q: For stripe case do we need to point to https://github.com/openebs/cstor-operators/pull/138? This means in case of stripe pool if the disk is lost/corrupted then we need to remove pool spec and create new pool spec and scale volume replicas in that pool.